lom/src/plugins/plugins_common/plugins_common.go

```diff
@@ -122,8 +122,10 @@ type PluginMetadata struct {
     StartedTime time.Time
     Pluginstage PluginStage // indicate the current plugin stage
     PluginId
-    mu                          sync.Mutex // Mutex to synchronize access to PluginStage field
-    PluginResponseRollingWindow            // rolling window of response times
+    mu                           sync.Mutex    // Mutex to synchronize access to PluginStage field
+    PluginResponseRollingWindow                // rolling window of response times
+    MaxPluginResponses           int           // max number of responses that plugin can send per anamolykey during last MAX_PLUGIN_RESPONSES_WINDOW_TIMEOUT
```

Comment:
Shouldn't these be entries be part of ActionCfg, so it can be sent to Plugin during init ?
Plugin & Mgr should have same numbers.
Please check with Goutham, how/where he gets his numbers from?

Reply:
We agreed to separate them. These are generic for all plugins. So that plugin manager will decide and manage timeouts independently.

Reply:
This should be made configurable

Reply:
sync with Prithvi

Reply:
TODO